.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.1
```

Comment:
rbenv wants it -- I can put it in .gitignore.

But we're already specifying 2.2.1 in https://github.com/projecthydra-labs/geo_concerns/pull/111/files/bd011649f94ff3537f9b36c6bb478bedc3e6f229#diff-8b7db4d5cc4b8f6dc8feb7030baa2478L2

So if `Gemfile` demands 2.2.1, then `.ruby-version` needs to match.  Maybe remove them both?